--- origsrc/gtkdatabox-0.9.1.1/examples/Makefile.am	2009-04-19 17:15:49.000000000 -0500
+++ src/gtkdatabox-0.9.1.1/examples/Makefile.am	2011-05-30 00:00:05.539711700 -0500
@@ -52,7 +52,6 @@ AM_CFLAGS			=\
 			-DG_DISABLE_DEPRECATED\
 			-DGDK_DISABLE_DEPRECATED\
 			-DGDK_PIXBUF_DISABLE_DEPRECATED\
-			-DGTK_DISABLE_DEPRECATED\
 			@LIBGLADE_CFLAGS@ \
 			`pkg-config gtk+-2.0 --cflags`
 
--- origsrc/gtkdatabox-0.9.1.1/gtk/Makefile.am	2011-05-29 23:40:45.975388400 -0500
+++ src/gtkdatabox-0.9.1.1/gtk/Makefile.am	2011-05-29 23:48:01.268285700 -0500
@@ -56,6 +56,5 @@ AM_CFLAGS			=\
 			-DG_DISABLE_DEPRECATED\
 			-DGDK_DISABLE_DEPRECATED\
 			-DGDK_PIXBUF_DISABLE_DEPRECATED\
-			-DGTK_DISABLE_DEPRECATED\
 			-DGTK_MULTIHEAD_SAFE=1\
 			`pkg-config gtk+-2.0 --cflags`
